libkbfs/interfaces.go

```diff
@@ -2037,11 +2041,18 @@ type initModeGetter interface {
 	IsTestMode() bool
 }
 
+type blockCryptVersioner interface {
+	// BlockCryptVersion returns the block encryption version to be used for
+	// new blocks.
+	BlockCryptVersion() kbfscrypto.EncryptionVer
```

Comment:
any reason to pass this around instead of passing the version? Will it be changable at runtime?

Reply:
Yeah, my plan is to phase this in with admins first, setting it in the init function based on your session's keybase ID.